mmdet/evaluation/metrics/coco_metric.py

```diff
@@ -8,6 +8,8 @@
 
 import numpy as np
 import torch
+from faster_coco_eval import COCO as FasterCOCO
```

Comment:
I think it's best to make this dependency optional, to prevent affecting the previous usage pattern.

Reply:
done